docs/config.md

```diff
@@ -231,3 +235,39 @@ You can specify multiple networks (or subnets) to connect your server to. To do
           - uuid: your_network_id
           - subnet_id: your_subnet_id
 ```
+
+## Tagging Instances
+Tags can be added to instances on startup by populating the tags array where it says `<Your Tags>`. You can remove this if you do not want to tag your instances. Please note that your Nova api must be at least mivroversion 2.52 to use this api! Please refer to the following usage example:
```

Comment:
https://developer.openstack.org/api-ref/compute/#server-tags-servers-tags
actually you can set 2.26 version by first create a VM then tag it

Reply:
A-ha, this is a good point. We should probably do this. Just set the version to 2.52 when `Tags` is present.

@iamemilio mind doing this? :)

Reply:
Did it, a bit different but I think it's a good start.